odr-dabmux.service

```diff
@@ -6,6 +6,9 @@ After=network-online.target sound.target
 
 [Service]
 Type=simple
+ExecStartPre=-/bin/mkdir -p /var/tmp/dabmux
+ExecStartPre=-/bin/mkfifo --mode=644 /var/tmp/dabmux/odr-dabmux-local-01.fifo
+ExecStartPre=-/bin/chown odr-dabmux:odr-dabmux /var/tmp/dabmux/odr-dabmux-local-01.fifo
 ExecStart=/usr/bin/odr-dabmux /etc/odr-dabmux/dab.mux
 User=odr-dabmux
 Group=odr-dabmux
```

Comment:
I think such a stop post hook config is rather limiting. I would like to keep the fifo around since it might still have connected clients (ie. dablin) that might well survive a restart of the mux.

Is there a reason for breaking that case? If we really need more that the current code does (ie. protect against a regular file overrunning a disk) we should split this into a separate systemd fifo socket config (also taking care of "our" dablin case).

I feel writing all of that is rather overkill given our current use case (local feedback) and our small user base (us). I'd be happy to to do (and properly test) that down the line when this has gotten more use.